db/migrations/20230202164851-create-salt-column.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Proposals', 'auth_password_salt', {
```

Comment:
Even though the additional migration is not _necessary_ I believe it's not a bad thing to have it there.

Since this PR establishes the structure of the files and folders, having two migrations adds a kind of documentation to the codebase, making `addColumn` and `removeColumn` operations available on the repository history.

But this is pretty subjective. What do you think?

Reply:
Since this is not in production anywhere, I prefer having have less migrations on the projects and flatten those migrations

Reply:
Flattened the migrations :+1: